diff --git a/.gitmodules b/.gitmodules
index 25142daddc6..8ae276efdda 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -5,7 +5,7 @@
 [submodule "gcc"]
 	path = gcc
 	url = https://gcc.gnu.org/git/gcc.git
-	branch = releases/gcc-13
+	branch = releases/gcc-14
 [submodule "glibc"]
 	path = glibc
 	url = https://sourceware.org/git/glibc.git
diff --git a/gcc b/gcc
index c891d8dc23e..cd0059a1976 160000
--- a/gcc
+++ b/gcc
@@ -1 +1 @@
-Subproject commit c891d8dc23e1a46ad9f3e757d09e57b500d40044
+Subproject commit cd0059a1976303638cea95f216de129334fc04d1
